package.json

```diff
@@ -4,27 +4,31 @@
   "version": "0.0.0",
   "description": "Robust, pluggable, & render agnostic framework for rapid delivery of modern web applications.",
   "scripts": {
+    "preinstall": "npx only-allow pnpm",
     "lint": "eslint scripts",
-    "lint:fix": "npm run lint -- --fix",
-    "lint:fix:all": "npm run lint:fix && lerna run lint:fix --stream",
-    "test": "lerna run test --stream",
-    "test:coverage": "lerna run test:coverage --stream",
-    "posttest": "npm run lint",
-    "build": "lerna run build",
+    "lint:fix": "pnpm run lint --fix",
+    "lint:fix:all": "pnpm run lint:fix && pnpm run -r lint:fix",
+    "test": "pnpm run -r test",
+    "test:coverage": "pnpm run -r test:coverage",
+    "posttest": "pnpm run lint",
+    "build": "pnpm run -r build",
     "align-packages": "node ./scripts/align-packages",
-    "clean": "npm run clean:packages && npm run clean:root",
-    "clean:packages": "lerna clean --yes",
+    "version": "changeset version",
+    "release": "changeset publish",
+    "changeset": "changeset",
+    "clean": "pnpm run clean:packages && pnpm run clean:root",
+    "clean:packages": "pnpm -r exec rimraf dist node_modules",
     "clean:root": "rimraf node_modules",
-    "docs": "lerna run docs && npm run generate-docs-index",
-    "docs-view": "npm run generate-docs-index && cd site && docusaurus start",
+    "docs": "pnpm run -r docs && pnpm run generate-docs-index",
+    "docs-view": "pnpm run generate-docs-index && cd site && docusaurus start",
     "generate-docs-index": "node scripts/generate-docs-index",
-    "postinstall": "npm run build",
+    "postinstall": "pnpm run build",
     "postpublish": "node scripts/mark-deprecated.mjs",
-    "publish": "lerna publish",
-    "publish:canary": "lerna publish --preid canary --dist-tag canary",
-    "publish:next": "lerna publish prerelease --preid next --dist-tag next --force-publish --exact",
-    "test:package": "lerna run test --stream --scope $*",
-    "typecheck:all": "lerna run typecheck --stream"
+    "publish": "pnpm changeset publish",
+    "publish:canary": "pnpm changeset version --snapshot canary && pnpm changeset publish --tag canary",
```

Comment:
I had to look up [snapshot docs](https://github.com/changesets/changesets/blob/main/docs/snapshot-releases.md). Sounds like an interesting for canary versions.